code/game/machinery/vending/vendor_types/antag/antag_predator.dm

```diff
@@ -131,6 +156,44 @@ GLOBAL_LIST_INIT(cm_vending_elder_yautja, list(
 /obj/structure/machinery/cm_vending/clothing/yautja/elder/get_listed_products(mob/user)
 	return GLOB.cm_vending_elder_yautja
 
+/obj/structure/machinery/cm_vending/clothing/yautja/young_blood
+	name = "\improper Yautja Young Hunting Gear Rack"
+	icon = 'icons/obj/items/hunter/pred_vendor.dmi'
+	icon_state = "pred_vendor_left"
+	req_access = list(ACCESS_YAUTJA_SECURE)
+	vendor_role = list(ERT_JOB_YOUNGBLOOD, JOB_PREDATOR)
+	show_points = FALSE
+	vendor_theme = VENDOR_THEME_YAUTJA
+
+/obj/structure/machinery/cm_vending/clothing/yautja/young_blood/can_access_to_vend(mob/user, display = TRUE, ignore_hack = FALSE)
+	if(!allowed(user))
+		if(display)
+			to_chat(user, SPAN_WARNING("Access denied."))
+			vend_fail()
+		return FALSE
+
+	if(LAZYLEN(vendor_role) && !vendor_role.Find(user.job))
+		if(display)
+			to_chat(user, SPAN_WARNING("This machine isn't for you."))
+			vend_fail()
+		return FALSE
+	return TRUE
```

Comment:
This is the exact same as `/obj/structure/machinery/cm_vending/clothing/yautja/can_access_to_vend`, redundant